Tests/ML/histopathology/utils/test_metrics_utils.py

```diff
@@ -57,6 +77,78 @@ def test_select_k_tiles() -> None:
     assert_equal_lists(bottom_tp, [(4, 1.0, [2, 1], [Tensor([0.31]), Tensor([0.33])]), (2, 0.7, [1, 4], [Tensor([0.10]), Tensor([0.13])])])
 
 
-def test_plot_scores_hist() -> None:
+def test_plot_scores_hist(test_output_dirs: OutputFolderForTests) -> None:
     fig = plot_scores_hist(test_dict)
     assert isinstance(fig, matplotlib.figure.Figure)
+    file = Path(test_output_dirs.root_dir) / "plot_score_hist.png"
+    resize_and_save(5, 5, file)
+    assert file.exists()
+    expected = full_ml_test_data_path("histo_heatmaps") / "score_hist.png"
+    # To update the stored results, uncomment this line:
+    # expected.write_bytes(file.read_bytes())
+    assert_binary_files_match(file, expected)
```

Comment:
Interesting. I never designed tests where I was asserting the binary equality of two pngs. I wonder how robust is this to minor changes in matplotlib or the platform in which the png is created. @ant0nsc what you think? In general happy to try this, thanks for implementing Harshita

Reply:
@vale-salvatelli , I was the one who suggested this to Harshita. We have been using this type of tests successfully for some time. Yes, rendering is platform specific, but we run all tests on Linux only now.